lib/src/backend.rs

```diff
@@ -458,7 +497,7 @@ pub trait Backend: Send + Sync + Debug {
     /// unnecessary resources.
     fn get_copy_records(
         &self,
-        paths: &[RepoPathBuf],
+        paths: Option<&[RepoPathBuf]>,
```

Comment:
nit: can you also update the doc string?
> If `paths` is empty include all paths

Ideally, this change could be moved to the first commit, but I don't mind.

Reply:
done